### PR TITLE
Fixed Supply Tokens

### PR DESCRIPTION
**Pull request by talekhinezh**
Add support for Fixed Supply Tokens along with a lot of refactoring.

### CODE REVIEW

File: examples/src/main/java/com/radixdlt/client/examples/RadixWalletExample.java [hunk 1]
```diff
@@ -38,28 +37,33 @@ public static void main(String[] args) throws Exception {
 			.getStatusUpdates()
 			.subscribe(System.out::println);
 
-
 		RadixApplicationAPI api = RadixApplicationAPI.create(radixIdentity);
```
Comment (flotothemoon) [inline examples/src/main/java/com/radixdlt/client/examples/RadixWalletExample.java]: <nitpicking> Should we capitalize it RadixApplicationApi instead for consistency? </nitpicking>

**Comment by flotothemoon** [comment]
I would suggest making the actions stand on their own - verbs in class names seem a bit awkward to me. So instead of TransferTokens we could call it TransferTokensAction etc

File: examples/src/main/java/com/radixdlt/client/examples/RadixWalletExample.java (hunk 1 above)
Comment (talekhinezh) [reply examples/src/main/java/com/radixdlt/client/examples/RadixWalletExample.java]: Not a fan of changing signatures at the moment as will probably change it down the line again as architecture gets more defined to something like `RadixApplication`

**Comment by talekhinezh** [comment]
@flotothemoon agreed.

**Pull request merged**